invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/state/types.js

```diff
@@ -1,6 +1,3 @@
 export const IS_LOADING = 'fetchLoanDetails/IS_LOADING';
 export const SUCCESS = 'fetchLoanDetails/SUCCESS';
 export const HAS_ERROR = 'fetchLoanDetails/HAS_ERROR';
-export const ACTION_IS_LOADING = 'performLoanAction/IS_LOADING';
```

Comment:
how is now the loan action handled?

Reply:
Since both loan actions and the other still update the same data and even if it's an action causing something we still want the component to be in a loading state so they all use the same types. @topless do you have something to add to why this decision was made?